src/views/AuthoritiesSearch/AuthoritiesSearch.js

```diff
@@ -113,6 +116,13 @@ const AuthoritiesSearch = ({
 
   const filterPaneVisibilityKey = getNamespace({ key: 'marcAuthoritiesFilterPaneVisibility' });
 
+  useEffect(() => {
```

Comment:
I think if you're not setting `isInitialMount.current` to `false` anywhere then each time this hook the condition will be true.
As an alternative, can we use `history.goBack` in `AuthorityQuickMarcEditRoute onClose`? It'll redirect back to previous page and keep the search url

Reply:
Hmm, you're right about not setting `isInitialMount.current` to false. There was some strange behavior after I tried to do it, so I went with the `history.goBack` approach. Thanks for the suggestion!